app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from __future__ import annotations
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list) -> int:
```

Comment:
Always try to provide full typing
```suggestion
def calculate_team_total_rating(players: list[Player]) -> int:
```